app/javascript/application.js

```diff
@@ -1,3 +1,5 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+import Rails from "@rails/ujs"
+Rails.start()
```

Comment:
I mentioned this elsewhere in a comment, but for the sake of it being here where the context is helpful, I have no real objection to this. If the only thing we're hoping for out of UJS is data-confirm, I might opt to write a stimulus controller for that.

Reply:
Yea, that’s true, it would be easy. I’ve been confused about UJS but I just did enough reading to finally understand — hotwire really is intended as a full replacement for it. In fact, it looks like they’ve added:

data-turbo-confirm=“are you sure?”
https://github.com/hotwired/turbo/pull/379

and even: data-turbo-method=“delete”

I’ll do the proper fix for this.

Reply:
Oh good find, I wasn't aware of that but it sure seemed like UJS had gone stale.